src/transaction.js

```diff
@@ -71,6 +72,35 @@ export class Transaction {
     });
   }
 
+  /**
+   * Add a signature to the transaction. Useful when a party wants to pre-sign
+   * a transaction but doesn't want to give access to their secret keys.
+   * @param {string} publicKey The public key of the signer
+   * @param {string} signature The base64 value of the signature XDR
```

Comment:
I think it doesn't work with base64 encoded signatures. Tests seem to add byte arrays instead of b64 encoded strings.

Reply:
Good catch, should be fixed now!